jbmc/regression/jbmc/instanceof2/test.desc

```diff
@@ -1,4 +1,4 @@
-KNOWNBUG
+CORE
```

Comment:
:pick: While your in the area and presumably understand the test - could you add a description to the bottom of it.